reference/batch-workflow-syntax.md

```diff
@@ -189,6 +204,172 @@ cache:
 
 **Expression contexts:** [`flow` context](batch-contexts.md#flow-context), [`params` context](batch-contexts.md#params-context), [`env` context](batch-contexts.md#env-context), [`tags` context](batch-contexts.md#tags-context), [`volumes` context](batch-contexts.md#volumes-context), [`images` context](batch-contexts.md#images-context).
 
+## `images`
+
+A mapping of image definitions used by the _batch_ workflow.
+
+If the specified image reference is available at the Neu.ro registry and the [`.force_rebuild`](batch-workflow-syntax.md#images-less-than-image-id-greater-than-force-rebuild) flag is not set, then Neu.ro Flow starts building the image from scratch. 
+If this flag is set, then the existing image from the registry will be used.
```

Comment:
Sorry, this is a little bit wrong.
If an image is not neuro registry image (ref doesn't start from `image:`) - the system will try to use ref as is.
If the flag is not set **and** image in registry - no build.
If the flag is set **or** image not in registry - build will happen.

Reply:
Fixed in the recent commit